pages/dkp/kommander/2.1/release-notes/2.1.1/index.md

```diff
@@ -77,6 +77,91 @@ The following services and service components have been upgraded to the listed v
 - traefik-forward-auth: 0.3.2
 - velero: 3.1.3
 
+## Known Issues 
+
+### Create cert-manager resources on attached clusters with cert-manager pre-installed
+
+If you are attaching a cluster that already has `cert-manager` installed, you need to manually create some cert-manager resources after attaching your cluster.
+
+For example, [Konvoy-created clusters that are self-managed][konvoy-self-managed] have `cert-manager` already installed to the `cert-manager` namespace.
+
+Check whether your cluster is self-managed by:
+
+```bash
+export KUBECONFIG=<kubeconfig-path>
+export WORKSPACE_NAMESPACE=<your-workspace-namespace>
+kubectl get kommandercluster -n $WORKSPACE_NAMESPACE
```

Comment:
Want to double check if this method can be described like this.

Reply:
Do you need some assist testing this?

Reply:
`kommandercluster` is a kommander resource that only exists on the management cluster

Reply:
gonna change it to check cert-manager pod instead as discussed above